app/src/main/res/values/styles.xml

```diff
@@ -305,7 +305,6 @@
         <item name="elevation">4dp</item>
         <item name="android:outlineAmbientShadowColor" tools:targetApi="p">@color/black</item>
         <item name="android:outlineSpotShadowColor" tools:targetApi="p">@color/black</item>
-        <item name="android:background">?attr/paper_color</item>
```

Comment:
Removing this attribute introduces significant regressions in the appearance of other Toolbars. e.g. in MainActivity, upon scrolling down. This attribute should be kept, until we refactor all other Toolbars to have the same background color.